api-reference/10 UI Widgets/dxFileUploader/3 Methods/abortUpload().md

```diff
@@ -3,7 +3,26 @@ id: dxFileUploader.abortUpload()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Cancels the file upload.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+---
+
+##### jQuery
+
+    <!-- tab: JavaScript -->
+    var uploadControl = $("#uploaderContainer").dxFileUploader("instance");
+
+    uploadControl.abortUpload()
+
+---
+
+[note]
+The **abortUpload** method works differently in **useForms**, **useButtons** and **instantly** [upload modes](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode):
+
+- **useForms**: The method is not supported in this mode.
+
+- **useButtons**: Cancels the file upload and makes the file ready for uploading.  
```

Comment:
Can you clarify what you mean by "makes the file ready"? Does the file become "available" for upload?

Reply:
Yes. The "upload" button/link appears under the file and it is available for upload.

Reply:
```suggestion
- **useButtons**: Cancels the file upload and makes the file available for upload.  
```